db_connect.ts

```diff
@@ -12,12 +12,13 @@ if (!MONGO_URI) {
   throw new Error('You must provide a MongoDB URI');
 }
 logger.info(`Connecting to: ${MONGO_URI}`);
+// TODO: Update mongoose to latest version
 const connect = () => {
   mongoose.connect(MONGO_URI, {
     useNewUrlParser: true,
     useUnifiedTopology: true,
-  });
+  } as any); // older version of mongoose that does not have the ConnectOptions type defined.
```

Comment:
@Mersho let's update this in a separate commit, to avoid having to write this comment